crates/ra_ide_api/src/syntax_highlighting.rs

```diff
@@ -11,6 +11,14 @@ pub struct HighlightedRange {
     pub tag: &'static str,
 }
 
+fn is_control_keyword(kind: SyntaxKind) -> bool {
+    match kind {
+        FOR_KW | LOOP_KW | WHILE_KW | CONTINUE_KW | BREAK_KW | IF_KW | ELSE_KW | MATCH_KW
```

Comment:
The new syntax for this is

```
match kind {
    T![for], T![loop], ...
}
```

Reply:
Is that better? :sweat_smile:

Updated.

Reply:
I think it is, but the beauty is in the eye of the beholder :)